docs/StardustDocs/topics/gettingStartedGradleAdvanced.md

```diff
@@ -128,6 +128,10 @@ dependencies {
     implementation("org.jetbrains.kotlinx:dataframe-jdbc:%dataFrameVersion%")
     implementation("org.jetbrains.kotlinx:dataframe-arrow:%dataFrameVersion%")
     implementation("org.jetbrains.kotlinx:dataframe-openapi:%dataFrameVersion%")
+    // Module needed to directly call functions that generate @DataSchema code from openapi specs
```

Comment:
OpenAPI

Reply:
I'd specify something that indicates this module is usually not needed. Something like "This artifact is only needed when you need to directly call functions that...". That conveys the message better than "Module needed to..." I think.

Reply:
Thanks. Since you implemented openapi, i wanted to see how you'd phrase it